__init__.py

```diff
@@ -72,6 +73,9 @@ def initialize(self):
             "datetime_skill") or "skill-ovos-date-time.openvoiceos"
         self.examples_enabled = 1 if self.settings.get(
             "examples_enabled", True) else 0
+
+        self.examples_prefix = self.settings.get("examples_prefix", True)
```

Comment:
this var should be removed, access self.settings directly when used, this ensures it will catch settings changes live without needing a reload

Reply:
I was just following existing code, but makes sense. I didn't know skill settings were updated immediately on file changes

Reply:
auto reload was introduced in version 0.0.4